### ISSUE TITLE
Handling a sitemap that contains pdfs

### ISSUE THREAD

**Issue by bjgordon** [pa11y/ci#10]
I would like to use pa11y-ci in a jekyll site. The easiest way seems to be using it with a sitemap, however the site has some pdf files which are included in the sitemap, and pa11y-ci gets an error when it tries to check these:
```
Error: Error opening url
   "http://localhost:4000/files/file.pdf" :
   undefined
```
I could possibly remove the pdf files from the sitemap, but AFAIK it's better to include them for search engines to index. Maybe pa11y-ci needs an option to exclude certain files from the sitemap e.g. with a regex? Or is there another way I can handle these?

**Comment by bjgordon** [comment]
Currently thinking my easiest option is to create another sitemap just for pa11y which does not include pdfs...

**Comment by rowanmanning** [comment]
Hi @bjgordon, yeah it'd be really useful to be able to exclude URLs from the sitemap. Thinking maybe a `--sitemap-exclude` flag. Would be happy to accept a PR, otherwise I can work on this some time soon

**Comment by andrewmee** [comment]
Hi @bjgordon I've just published a new version 1.1.0 which includes a `--sitemap-exclude` command-line option.  Hoping this helps you simplify things in your setup :)

**Comment by bjgordon** [comment]
Awesome, thanks @andrewmee
